shedskin/shedskin
# tests2 progress

Further work on `tests2` folder: key thing now is that all tests in `tests2` pass for `pytest`, and (cmake) cpp-exe, and cpp-ext (python extension) cases.

To test using `pytest`:

```bash
cd tests2
pytest
```

To test using cmake for cpp-exe:

```bash
cd tests2
./runtests.py -c
```

To test using cmake for cpp-ext (python extension):

```bash
cd tests2
./runtests.py -ce
```

The next iteration should be to add windows support for the cmake test-runner and also to test both cpp-exe and cpp-ext cases in the same test run instead of in separate test runs.
